src/app/services/article.service.ts

```diff
@@ -13,11 +13,14 @@ export class ArticleService {
     "stories",
     "church-construction-complete",
     "is-peter-rock",
-    "why-not-catholic"
+    "why-not-catholic",
+    "church-construction-complete",
+    "is-peter-rock",
+    "why-not-catholic",
   ];
-  public defaultImageURI = "https://t4.ftcdn.net/jpg/02/97/78/03/360_F_297780357_pK8VCA7wctbTFusAGiCfcoxbJLRwC9Bs.jpg";
-  public NEW_LABEL:string = "[ new ]";
-  public BASE_FIRESTORE: string = "https://firestore.googleapis.com/v1";
+  public readonly defaultImageURI = "https://t4.ftcdn.net/jpg/02/97/78/03/360_F_297780357_pK8VCA7wctbTFusAGiCfcoxbJLRwC9Bs.jpg";
```

Comment:
I think these 3 should ben in a Constants Class and I probably should have named `defaultImageURI` as `DEFAULT_IMAGE_URI` since it contains "magic" (hardcoded) content.

Reply:
Agreed. I made them read only because that's exactly what they are. Without read only or constant there's the chance they are intended to be mutated or modified in some way.